wg-nodeless/README.md

```diff
@@ -24,5 +24,17 @@ Explore and ultimately propose a nodeless design that can extend a kubernetes cl
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-nodeless)
 
 <!-- BEGIN CUSTOM CONTENT -->
-The proposal and goals for the nodeless working group are outlined [here](https://docs.google.com/document/d/1Y1GEKOIB1u5P06YeQJYl9WVaUqxrq3fO8GZ7K6MUGms/edit)
+The goals for the working group are as follows: 
+Phase 1
+1. Define what nodeless Kubernetes means to users, by exploring the realm of use cases 
+2. Design a user experience around nodeless 
+3. Create reference architectures for a nodeless Kubernetes and define clear behaviors for how a nodeless Kubernetes should act
+
+Phase 2
```

Comment:
Can this be omitted since it presupposes the outcome of phase one? If the goal is inquiry into a solution, that should be the deliverable. Phase 2 cannot be accomplished by a working group. The design from phase one will require consensus building in the affected sub-projects OWNERS in order to get those code changes in (_if applicable_). WGs do not own code, do not have any vested authority, and are intended to problem solve cross-SIG concerns. The problem is undefined until the work of phase one is complete by design of the WG itself.

Reply:
@jdumars are you asking us to delete phase two?

Reply:
I like the new summary sentence Ria. 

The output of the working group should be documentation that explores the domain and comes to detailed recommendations about the alternatives that present themselves. In order to do that, the group will need to:
- define the problem space
- examine and document user stories in terms of business value
- explain the limitations of the status quo Eg. Why is namespace support not enough?
- state clear assumptions about existing CaaS APIs and what limitations that puts on a design
- investigate the possible solutions, propose potential designs and speculate on the impact

In that respect, I think (5) is important to keep in scope because any analysis of approaches needs to account for limitations and user impact. (7) matters because a solution can't be credible unless all assumptions are made explicit, but could be rephrased as "document assumptions" rather than a detailed API definition. (4) and (6) seem unnecessary, because I would expect that any upstream changes or compliance questions would be dependent on the solution being accepted.

Reply:
I actually don't mind this section.  My reading is that "propose" != "implement".  Proposals from a WG are fine.

I would drop 7, though.  Kubernetes is concerned with the UX and API - any platform that can meet those needs is fine.  It's not our place to define what providers need to do.